### ISSUE TITLE
Autocomplete attributes should be added to appropriate fields

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2245]
**Describe the solution you'd like**
Autocomplete attributes should be added to appropriate fields for accessibility

**Additional context**
List of known input types that should have attributes attached https://www.w3.org/TR/WCAG21/#input-purposes

**What is the acceptance criteria/definition of done?**
List of known input types that correspond to attributes:
- name
- given-name 
- additional-name
- family-name
- new-password
- current-password
- organization
- address-line1
- address-line2
- postal-code
- bday
- bday-day
- bday-month
- bday-year
- tel
- email

**QA Review Instructions**
For each form field that collects information about the user and corresponds to an autocomplete field described in WCAG 2.1 7. Section 7: Input Purposes for User Interface Components, check the following:
- The form field has a valid and well-formed autocomplete attribute and value pair.

**Comment by slowbot** [comment]
@emilyjablonski For your review
